app/models/fail.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Fail < ApplicationRecord
+  include Fail::Request
+
+  belongs_to :user, optional: true
+
+  validates :data, presence: true
```

Comment:
```suggestion
  validates :data, presence: true

  belongs_to :user, optional: true
```
Solo el cambio de orden.

Reply:
buenas practicas jajajaja